requirements.txt

```diff
@@ -16,3 +16,4 @@ repoze.lru
 gevent-websocket==0.9.4
 sphinx
 bumpversion
+cachetools>=2.0.0<3.0.0
```

Comment:
> Is it to avoid any unintentional code behaviour changes in the case a new cachetools major version is released?

Yes, also, I'm assuming proper semantic versioning, it might not be the case.